### PR TITLE
refactor: separate code from compiled artifacts

### PR DESCRIPTION
**Pull request by severo**
See https://github.com/severo/observable-to-standalone/issues/5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
   "description": "Standalone version of the Spherical contours notebook",
   "main": "index.js",
   "scripts": {
-    "data": "node bin/download-data.js",
-    "build": "rollup -c",
+    "clean": "rm -rf ./node_modules ./public ./vendor",
```
Comment (severo) [inline package.json:7]: @Fil Please comment if it's OK for you... It's always somewhat touchy to use automate "rm -rf"

Comment (Fil) [reply package.json:7]: I think it's ok

**Comment by severo** [comment]
I will propose a new one

**Comment by severo** [comment]
Updated. Please review.

**Pull request closed without merging**